roles/ceph-common/tasks/installs/debian_custom_repository.yml

```diff
@@ -1,7 +1,12 @@
 ---
+- name: add an apt signing key custom repository
+  apt_key:
+    url: "{{ ceph_custom_key }}"
+    state: present
```

Comment:
Could you add the following lines ?
```yaml
  register: result
  until: result is succeeded
```

Reply:
will do

Reply:
Do I have to wait jenkins jobs finished or I can just commit n push new changes ?

Reply:
You can push new changes before the jenkins jobs are finished.

Reply:
done